build-tools/packages/build-tools/src/fluidBuild/fluidUtils.ts

```diff
@@ -145,32 +145,36 @@ const configExplorer = cosmiconfigSync(configName, {
 });
 
 /**
- * Set to true when the default config is returned by getFluidBuildConfig so that repeated calls to the function don't
- * result in repeated searches for config.
+ * Contains directories previously used to start search but did find an explicit fluidBuild config file.
```

Comment:
I think this is what we wanted?
```suggestion
 * Contains directories previously used to start search but where we didn't find an explicit fluidBuild config file.
```

Reply:
Nope. we add the `searchDir` in `defaultSearchDir` when `configExplorer.search` returns `undefined` (i.e. not found)

Reply:
Gah... I read the diff backwards... you are right.